pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -3106,6 +3115,42 @@ public bool ExportChromatograms(string chromatogramsFile, bool precursors, bool
             return true;
         }
 
+        /// <summary>
+        /// Export a spectral library (.blib) file from the document
+        /// </summary>
+        /// <param name="specLibFile"> File path to export the spectral library to</param>
+        /// <returns></returns>
+        public bool ExportSpecLib(string specLibFile)
+        {
+            _out.WriteLine(Resources.SkylineWindow_ShowExportSpectralLibraryDialog_Exporting_spectral_library__0____, specLibFile);
+            if (Document.MoleculeTransitionGroupCount == 0) // The document needs at least one peptide precursor
+            {
+                _out.WriteLine(Resources.CommandLine_ExportSpecLib_Error__The_document_must_contain_at_least_one_peptide_precursor_to_export_a_spectral_library);

```

Comment:
Does this get automatically changed to talk about one molecule if the document is a small molecule document? If so, mention that in the comment. If not, we need to fix that.

Reply:
I changed the error message from "peptide precursor" to simply "precursor" to address this, and updated the comment